app/models/document.rb

```diff
@@ -56,6 +56,7 @@ def self.create_initial(content_id: SecureRandom.uuid,
                          document_type_id: document_type_id,
                          created_by: user)
 
+      tags.each { |_, v| v.reject!(&:empty?) }
```

Comment:
Agreed. Reason we have this line is because we needed to filter the content coming from the publishing API, which currently must store a nil organisation as an empty string rather than nil. If I can move it to the above I'd feel much happier too.

Reply:
We don't actually get content from the Publishing API that is stored, it's all user input.